src/components/Button/Button.scss

```diff
@@ -1,14 +1,20 @@
-@import "../../../../scss/variables";
+@import "../../scss/variables";
 
 .Button {
   background-color: $gray;
+  border: none;
   border-radius: 20px;
+  box-shadow: -2px 4px 4px
+  rgba(0, 0, 0, 0.17);
   color: $white;
   cursor: pointer;
   font-size: $font-md;
   height: 39px;
+  line-height: $line-height-sm;
   margin: 5px;
   outline: none;
-  padding: 0 15px;
+  padding: 0 30px;
   text-transform: uppercase;
-}
\ No newline at end of file
+  -webkit-appearance: button;
```

Comment:
I believe it should be none to not style it in any special way, need to check it later. Therefore, it is desktop app, we won't be displaying in the browser so I don't know if we need it at all or how it will be behaving on different operating systems